modules/340-monitoring-kubernetes/monitoring/prometheus-rules/kubernetes/ebpf-exporter.yaml

```diff
@@ -36,8 +36,8 @@
     annotations:
       plk_protocol_version: "1"
       plk_markup_format: "markdown"
-      plk_create_group_if_not_exists__d8_ebpf_exporter_malfunctioning: "EbpfExporterKernelNotSupportedGroup,tier=cluster,prometheus=deckhouse,kubernetes=~kubernetes"
-      plk_grouped_by__d8_ebpf_exporter_malfunctioning: "EbpfExporterKernelNotSupportedGroup,tier=cluster,prometheus=deckhouse,kubernetes=~kubernetes"
+      plk_create_group_if_not_exists__d8_ebpf_exporter_malfunctioning: "EbpfExporterKernelNotSupportedGroup,tier=cluster, node={{ $labels.node }},prometheus=deckhouse,kubernetes=~kubernetes"
```

Comment:
Please update the expression to include the `node` label in annotations. The updated expression should look like this:

```promql
min by (job, node) (ebpf_exporter_btf_support_unavailable_in_kernel{job="ebpf-exporter"}) == 1
```

Reply:
Oh, didn't know that we need similar labels in expression too.
Fixed!